lib/MusicBrainz/Server/Entity/URL/NicoNicoVideo.pm

```diff
@@ -17,6 +15,7 @@ no Moose;
 =head1 COPYRIGHT AND LICENSE
 
 Copyright (C) 2018 MetaBrainz Foundation
+		  (C) 2018 Theodore Fabian Rudy
```

Comment:
Nit-picking: Please insert `Copyright` ahead of `(C)` just like in the above, and add a blank line after your name so as to separate it from the license text.